lib/chromedriver2/helper/google_code_parser.rb

```diff
@@ -17,6 +18,7 @@ def downloads
         doc = Nokogiri::XML.parse(source)
         items = doc.css("Contents Key").collect {|k| k.text }
         items.reject! {|k| !(/chromedriver_#{platform}/===k) }
+        items.sort! { |k1, k2| k1.match(VERSION_MATCH)[1].to_i <=> k2.match(VERSION_MATCH)[1].to_i }
```

Comment:
this could be replaced with something more intelligent that also validates the first digit